typst.toml

```diff
@@ -7,3 +7,4 @@ license = "MIT"
 description = "Beautifully typeset algorithms."
 repository = "https://github.com/platformer/typst-algorithms"
 keywords = ["algorithm", "pseudocode", "code"]
+exclude = ["test", "examples/seperated"]
```

Comment:
This line isn't actually necessary. I only ever upload the minimally necessary files when updating the package.